app/controllers/admin/settings/base_controller.rb

```diff
@@ -24,6 +40,8 @@ def extra_authorization_and_confirmation
       end
 
       def confirmation_text_valid?
+        return true unless params.key?(:confirmation)
```

Comment:
The RFC calls for NOT using the Confirmation Text field in Config updates, so I added this guard. We still want the `confirmation_text_valid?` check in place for use within the rest of the Admin portal.